chainer/functions/normalization/batch_normalization.py

```diff
@@ -135,11 +152,19 @@ def backward(self, x_orig, gy):
         ggamma = (gy * self.x_hat).sum(axis=(0, 2), keepdims=True)
         self.ggamma += ggamma
 
-        coeff = self.gamma / self.std
-        gbeta /= m
-        ggamma /= m
+        if cuda.get_array_module(*x_orig) == numpy:
+            coeff = self.gamma / self.std
+            gbeta *= inv_m
+            ggamma *= inv_m
+            gx = coeff * (gy - self.x_hat * ggamma - gbeta)
```

Comment:
How about `gx = coeff * (gy - (self.x_hat * ggamma - gbeta) / m)`?
I think `inv_m` is not requried.